html final/styles.css

```
@@ -0,0 +1,853 @@
+@import url('https://fonts.google.com/specimen/Varela+Round#standard-styles');
+
+*{
+    font-family:Varela Round;
+
+}
+
+html{
+    scroll-behavior: smooth;
+
+}
+
+body{
+   width:100%;
+   height:1000px;
```

leave an empty space after : 
```suggestion
   width: 100%;
```